web/modules/custom/farm_page/templates/farm-homepage.html.twig

```diff
@@ -1,8 +1,16 @@
+{% set farm_welcome = "WELCOME TO " ~ data.farm.name ~ "!" %}
 {% if is_exist == true %}
-{{ store.field_description.value }}
-	{% for image in store_images %}
-	<img src={{image}}>
-	{% endfor %}
-{% else %}
-{{ message }}
+	<div class="intro-farm">
+		<div class="farm-name">
+			<h3>{{farm_welcome|upper}}</h3>
```

Comment:
Be consistent, put all properties to farm.
Eg: farm.name, farm.image, farm.description

Reply:
updated